simulator/virtual_machine.go

```diff
@@ -1701,8 +1701,16 @@ func (vm *VirtualMachine) CloneVMTask(ctx *Context, req *types.CloneVM_Task) soa
 			pool = vm.ResourcePool
 		}
 	}
+
+	DestHost := vm.Runtime.Host
```

Comment:
Non-exported variable names should start with lowercase, can you rename to `destHost` ?

Reply:
Commit amended ;-)